cornflow/endpoints/execution.py

```diff
@@ -171,17 +172,28 @@ def get(self, idx):
 
         self.user_id, self.admin, self.super_admin = Auth.return_user_info(request)
         execution = ExecutionModel.get_one_execution_from_user(self.user_id, idx)
-        if execution.finished:
+        if execution.state == 1:
             return {'status': 'finished'}, 200
         dag_run_id = execution.dag_run_id
         if not dag_run_id:
+            execution.state = -1
```

Comment:
here, I would use EXEC_STATE_UNKNOWN instead of "ERROR" because it did not even start the run. We can maybe create an error specific to that too (EXEC_STATE_ERROR_START) or something.